components/common/CardList/CardList.tsx

```diff
@@ -22,11 +22,7 @@ function CardList({ items, ...rest }: Props) {
       <ul className={styles.list}>
         {items.map((item) => (
           <li key={item.id}>
-            <Card
-              item={item}
-              {...rest}
-              onClick={() => handleClick(`${item.url}`)}
-            />
+            <Card item={item} onClick={() => handleClick(item.url)} {...rest} />
```

Comment:
지금 코드를 보면 ...rest에 Partial<UseModal> 값이 들어있는데요!
...rest라는 문만 봤을 때 이 맥락을 이해하기가 어려운 것 같아서 이런식으로 표현해보면 어떨까 싶네요! 
```suggestion
            <Card item={item} onClick={() => handleClick(item.url)} {...useModalRest} />
```

Reply:
근데 사실 써놓고 보니까 useModalRest가 좀 별로긴 한 것 같은데요!
모달을 띄우기 위해 모달에 대한 정보를 CardList가 props로 받지 않는 방법은 없을지 고민해봐도 좋을 것 같네요!